packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -726,6 +728,13 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
       ImmediateSchedulerPriority,
       performSyncWorkOnRoot.bind(null, root),
     );
+  } else if (
```

Comment:
GitHub won't let me comment on the lines above, but need to change 

```js
if (existingCallbackNode !== null) {
```

to check the priority instead, like we discussed out-of-band:

```js
if (existingCallbackPriority !== NoPriority) {
```

That will prevent it from scheduling a separate microtask per individual update

Reply:
Ah yeah, I still had that in my notes. Will do.